packages/rum/test/specs/apm-base.spec.js

```diff
@@ -154,29 +179,6 @@ describe('ApmBase', function() {
     expect(tr.name).toBe('ZoneTransaction')
   })
 
-  it('should instrument xhr when not active', function(done) {
```

Comment:
We still instrument XHR even if not active! patching is called in the `bootstrap.js` file